index.js

```diff
@@ -15,15 +15,17 @@ module.exports = postcss.plugin('postcss-nest-atrules', function() {
         const theRule = rule.clone()
         const theAtRule = atRule.clone()
         const parent = rule.parent
-        if (parent.type == 'atrule' && parent.name !== atRule.name) {
+        if (parent.type === 'atrule' && parent.name !== atRule.name) {
           const theParentAtRule = parent.clone()
           theParentAtRule.nodes = theRule.nodes
           theAtRule.nodes = [theParentAtRule]
         } else {
           theAtRule.nodes = theRule.nodes
         }
-        if (existingRules.hasOwnProperty(rule.selector)) {
-          existingRules[rule.selector].append(theAtRule)
+        if (
+          Object.prototype.hasOwnProperty.call(existingRules, rule.selector)
+        ) {
+          existingRules[rule.selector].nodes.push(theAtRule)
```

Comment:
@ai can you explain me why `existingRules[rule.selector].append(theAtRule)` doesn't work?

Reply:
Sorry, have no idea.